containers/ddev-webserver/Dockerfile

```diff
@@ -5,7 +5,6 @@
 ### and ddev-webserver-* (For DDEV local Usage)
 FROM ddev/ddev-php-base:20240723_stasadev_n_install_auto as ddev-webserver-base
```

Comment:
I guess this means you decided not to use the v1.23.4 tag for ddev-php-base, which is good for today's activities.